### DISCUSSION TITLE
About Why Alluxio doesn't accelerate the Trino query

### DISCUSSION THREAD

**Discussion by YuanYunjing** [trinodb/trino#13867] [General]
After integrating Trino with Alluxio, I finded the query speed is not faster than the query of Trino and HDFS. The data is indeed cached in Alluxio. Only when I reboot Trino, the query will be slow, that's the only change in query speed. So does anyone know why it shows like this?

Thank you!

**Comment by findepi** [comment]
That's more a question to Alluxio. cc @apc999 

BTW Trino comes with a builtin caching solution, see https://trino.io/docs/current/connector/hive-caching.html
